flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -451,13 +451,14 @@ public class NettyShuffleEnvironmentOptions {
                     .withDescription(
                             "Maximum backoff in milliseconds for partition requests of input channels.");
 
-    /** The timeout for partition request notifier. */
+    /** The timeout for partition request listener in result partition manager. */
     @Documentation.Section(Documentation.Sections.ALL_TASK_MANAGER_NETWORK)
-    public static final ConfigOption<Long> NETWORK_REQUEST_NOTIFY_TIMEOUT =
-            key("taskmanager.network.partition-request-notify-timeout-ms")
-                    .longType()
-                    .defaultValue(10000L)
-                    .withDescription("Timeout in milliseconds for partition request notifier in result partition manager.");
+    public static final ConfigOption<Integer> NETWORK_REQUEST_LISTENER_TIMEOUT =
+            key("taskmanager.network.partition-request-listener-timeout-ms")
```

Comment:
rename to `taskmanager.network.partition-request-timeout`, and change the type to `ConfigOption<Duration>`?